src/main/java/io/cdap/plugin/http/source/streaming/HttpStreamingSourceConfig.java

```diff
@@ -36,8 +36,8 @@ public class HttpStreamingSourceConfig extends BaseHttpSourceConfig {
   @Macro
   protected Long maxPagesPerFetch;
 
-  protected HttpStreamingSourceConfig(String referenceName) {
-    super(referenceName);
+  protected HttpStreamingSourceConfig(String referenceName, String authType, String oauth2Enabled) {
+    super(referenceName, authType, oauth2Enabled);
```

Comment:
This change is not correct and it should be resolved by the correct builder class creation in the parent class.